models/migrations/v1_21/v276.go

```diff
@@ -118,7 +123,9 @@ func migratePushMirrors(x *xorm.Engine) error {
 
 	for {
 		var mirrors []PushMirror
-		if err := sess.Limit(limit, start).Find(&mirrors); err != nil {
+		if err := sess.Select("push_mirror.id, push_mirror.repo_id, push_mirror.remote_name, push_mirror.remote_address, repository.owner_name as repo_owner, repository.name as repo_name").
+			Join("INNER", "repository", "repository.id = push_mirror.repo_id").
```

Comment:
If we are already at it, can we also use
```suggestion
			Join("INNER", "repository", "repository.id = push_mirror.repo_id").
			Where("push_mirror.remote_address IS NULL")
```
?
That way, the migration at least becomes symmetric.

Reply:
Since remote_address is not an index column and the second update will not affect anything, I don't think it's necessary for this change.